### PR TITLE
Vote 597/recently moved accordion bn

### PR DESCRIPTION
**Pull request by SamiraMSadat**

### CODE REVIEW

**Comment by SamiraMSadat** [comment]
Same issue as https://github.com/usagov/vote-gov/pull/723

**Comment by rayestrada** [comment]
@SamiraMSadat there was conditional logic added to the template to hide the empty accordion item for missing translations. Please merge in the work on the `feature/recently-moved-accordion` branch.

File: layouts/index.html [hunk 1]
```diff
@@ -25,7 +25,7 @@
           <div id="skip-target" class="tablet:grid-col-8 right-shift">
             <section class="register align-bottom">
               <header>
-                <h1 class="main-heading" data-test="main-header">{{ $translation.homepage.header | safeHTML }}</h1>
+                <h1 class="main-heading" data-test="main-header">{{ $translation.homepage.header }}</h1>
```
Comment (rayestrada) [inline layouts/index.html:28]: @SamiraMSadat please revert this file change.

**Review by rayestrada — CHANGES_REQUESTED** [review]

File: layouts/index.html (hunk 1 above)
Comment (SamiraMSadat) [reply layouts/index.html:28]: Fixed

**Review by rayestrada — DISMISSED** [review]

**Review by rayestrada — APPROVED** [review]

**Pull request merged**